amps/ags/rm-community/rm-community-repo/pom.xml

```diff
@@ -578,12 +578,11 @@
                       <images>
                          <image>
                             <!-- Quay image -->
-                            <name>${image.name}:${image.tag}</name>
-                            <registry>${image.registry}</registry>
+                             <name>${image.registry}/${image.name}:${image.tag}</name>
```

Comment:
Whitespace change made tags not aligned.

Reply:
Done